bin/getmaxmind.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const fs = require('fs');
+const http = require('http');
+const targz = require('targz');
+
+const URL = 'http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz';
+const DB_DIR = `${__dirname}/../db`;
+const GZ_FILE = `${DB_DIR}/GeoLite2-City.tar.gz`;
+const DB_FILE = `${DB_DIR}/GeoLite2-City.mmdb`;
+
+// check for existing file
+try {
+  fs.mkdirSync(DB_DIR);
```

Comment:
In more current versions of node, `exists` is deprecated - they want you to use `stat` or something else and catch the error.  Will change to do a better job catching explicit error codes.